tests/e2e/ctl_v3_member_test.go

```diff
@@ -50,17 +50,7 @@ func TestCtlV3MemberRemoveClientAutoTLS(t *testing.T) {
 func TestCtlV3MemberRemovePeerTLS(t *testing.T) {
 	testCtl(t, memberRemoveTest, withQuorum(), withNoStrictReconfig(), withCfg(*e2e.NewConfigPeerTLS()))
 }
-func TestCtlV3MemberAdd(t *testing.T)      { testCtl(t, memberAddTest) }
-func TestCtlV3MemberAddNoTLS(t *testing.T) { testCtl(t, memberAddTest, withCfg(*e2e.NewConfigNoTLS())) }
-func TestCtlV3MemberAddClientTLS(t *testing.T) {
-	testCtl(t, memberAddTest, withCfg(*e2e.NewConfigClientTLS()))
-}
-func TestCtlV3MemberAddClientAutoTLS(t *testing.T) {
-	testCtl(t, memberAddTest, withCfg(*e2e.NewConfigClientAutoTLS()))
-}
-func TestCtlV3MemberAddPeerTLS(t *testing.T) {
-	testCtl(t, memberAddTest, withCfg(*e2e.NewConfigPeerTLS()))
-}
+func TestCtlV3MemberAdd(t *testing.T)           { testCtl(t, memberAddTest) }
 func TestCtlV3MemberAddForLearner(t *testing.T) { testCtl(t, memberAddForLearnerTest) }
```

Comment:
Isn't this also covered?

Reply:
The underlying implementation of `TestCtlV3MemberAdd` and `TestCtlV3MemberAddForLearner` use `member add` and `member add --learner` command.

I thought it would be valuable to keep them in the e2e testing, to cover the real command usage scenarios.

Do we need to keep them?

Reply:
Let me know if I understood you. New framework runs e2e test, but runs etcdctl only with `--format=json`, so we never test `etcdctl member add`, but `etcdctl member add --format=json`?.

Reply:
Oh, I've got the point. 

I thought our common framework doesn't run e2e test. Now I know it does.

I'm going to remove these redundant functions.

Reply:
> I thought our common framework doesn't run e2e test. Now I know it does.

Common framework is an interface for testing that covers both e2e and integration. Tests in `tests/common` that use common framework will be automatically run as both e2e and integration tests. Write a test once and double the benefit. 

Instead of having two separate frameworks and redundant tests, we want to just have one framework with shared tests. For now we are migrating `tests/e2e` directory, however if you want you can look into `tests/integration` directory and check if some test scenarios are already covered and can be removed.

It's a good point that we don't test exact output of `etcdctl member add`, however that's much lower on our priority than testing e2e process.